shared/chat/conversation/messages/attachment/file/index.js

```diff
@@ -40,6 +41,13 @@ class FileAttachment extends React.PureComponent<Props> {
               {this.props.hasProgress && <Kb.ProgressBar ratio={this.props.progress} />}
             </Kb.Box>
           )}
+          {!!this.props.errorMsg && (
+            <Kb.Box style={styles.progressContainerStyle}>
+              <Kb.Text type="BodySmall" style={styles.fail}>
+                {this.props.errorMsg}
```

Comment:
Why not just have some static text in here saying "Failed to download attachment, please retry" instead of whatever the service is going to produce.

Reply:
Depends what happened whether retrying will help. Won't help if you're offline or your network is dead or the directory doesn't exist.

Reply:
It is not likely that whatever error the server plumbs up is going to give users any kind of useful information anyway, certainly will be tricky to parse it to figure out if retrying is worth their time. Worst case is they try again and it doesn't work again.

Reply:
ok